### PR TITLE
Cumulus 3861 - Dashboard: Replace Enzyme with React Testing Library(RTL)

### PR DESCRIPTION
**Pull request by vindigo**
**Summary:** Summary of changes
Project: CUMULUS-3861: Dashboard: Replace Enzyme with React Testing Library(RTL)
Reason:  Removed references to Enzyme and replaced them with React compliant testing components.

Addresses [CUMULUS-3861: Dashboard: Replace Enzyme with React Testing Library(RTL)
https://bugs.earthdata.nasa.gov/browse/CUMULUS-3861

## Changes

* Upgraded existing Enzyme test to React Testing LIbrary
* All test assertions in the test scripts are now passing without reference to Enzyme. Total of about 29 test assertions among these test scripts.

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: test/components/executions/execution-status.js [hunk 1]
```diff
@@ -1,15 +1,53 @@
+////*************************************************************************************************************************************
```
Comment (Jkovarik) [inline test/components/executions/execution-status.js:1]: We should probably attempt to include this information in our commit/PR log and not attempt to maintain in-line comments re: edit reasons.    At a minimum, this warrants a team norms conversation.

**Review by Jkovarik — CHANGES_REQUESTED** [review]
Request removal of, or discussion re: change comment blocks prior to merging.

File: localdashboard_cumulus_std.bash [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (Jkovarik) [inline localdashboard_cumulus_std.bash:1]: Nit: consistency - we're using .sh for most scripts in this and Core - there's no reason we can't use .bash to be more clear in the filename. 

Nit: Please consider *not* leaving scripts in the root of the project.

File: test/components/footer/version.js [hunk 3]
```diff
@@ -103,51 +101,53 @@ test('Dashboard Version is shown in the footer', function (t) {
...
+
+ const { container } = render(
+   <Footer
+     api={api}
+     apiVersion={apiVersion}
+     cmrInfo={cmrInfo}
+   />
+ );
+
+ // const footerLeftSideLinks = footerWrapper.find('[className="footer__links"] div');
```
Comment (Jkovarik) [inline test/components/footer/version.js:135]: Nit: remove

File: test/components/executions/execution-status.js (hunk 1 above)
Comment (BWex-NASA) [reply test/components/executions/execution-status.js:1]: These were my comments as part of an effort to begin some sort of test documentation. I found a lack of documentation on what the test was trying to accomplish, as well as how to even call it to run the test script file. I agree we should have a conversation on where this type of information can be stored/shared. Thanks @Jkovarik.

**Pull request merged**